apps/docs/components/mdx-components/generic.tsx

```diff
@@ -7,7 +7,7 @@ export const UnorderedList = (props: any) => {
 }
 
 export const OrderedList = (props: any) => {
-	return <ol {...props} />
+	return <ol className={props.className} {...props} />
```

Comment:
you don't need to explicitly add this because the `...props` will effectively do this :)